gdal-sys/src/lib.rs

```diff
@@ -6,3 +6,27 @@
 
 include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 include!(concat!(env!("OUT_DIR"), "/docs_rs_helper.rs"));
+
+impl Default for OGREnvelope {
+    fn default() -> Self {
+        Self {
+            MinX: 0.0,
+            MaxX: 0.0,
+            MinY: 0.0,
+            MaxY: 0.0,
+        }
+    }
+}
+
+impl Default for OGREnvelope3D {
+    fn default() -> Self {
+        Self {
+            MinX: 0.0,
+            MaxX: 0.0,
+            MinY: 0.0,
+            MaxY: 0.0,
+            MinZ: 0.0,
+            MaxZ: 0.0,
+        }
+    }
+}
```

Comment:
Is this controversial? Can we get bindgen to do it for us (on selected structs)?

Reply:
I don't think so, looks good to me.